src/arctic3d/modules/pdb.py

```diff
@@ -469,22 +394,27 @@ def fetch_pdb_files(pdb_to_fetch):
     Returns
     -------
     validated_pdbs : list
-        list of tuples (pdb_file, hit)
+        list of tuples (pdb_file, cif_file, hit)
     """
-    validated_pdbs = []
+    validated_pdb_and_cifs = []
     valid_pdb_set = set()  # set of valid pdb IDs
     for hit in pdb_to_fetch:
         pdb_id = hit["pdb_id"]
-        pdb_fname = f"{pdb_id}.pdb"
-        if pdb_fname not in os.listdir():
-            pdb_f = fetch_pdb(pdb_id)
+        chain_id = hit["chain_id"]
+        pdb_fname = f"{pdb_id}-{chain_id}.pdb"
+        cif_fname = f"{pdb_id}_updated.cif"
+        if cif_fname not in os.listdir():
+            cif_f = fetch_updated_cif(pdb_id, cif_fname)
+            pdb_files = convert_cif_to_pdbs(cif_f, pdb_id, uniprot_id)
+            log.info(f"converted cif to pdb files: {pdb_files}")
         else:
-            pdb_f = Path(pdb_fname)
-        if pdb_f is not None:
-            validated_pdbs.append((pdb_f, hit))
+            cif_f = Path(cif_fname)
+        pdb_f = Path(pdb_fname)
+        if pdb_f.exists():
+            validated_pdb_and_cifs.append((pdb_f, cif_f, hit))
             if pdb_id not in valid_pdb_set:
                 valid_pdb_set.add(pdb_id)
-    return validated_pdbs
+    return validated_pdb_and_cifs
```

Comment:
Here if there is you cannot find any pdbs it will return an empty list to `validate_api_hit` that will go as an argument for `get_maxint_pdb` which will return `pdb_f/cif_f is None` which will raise an error in the `cli.py` i'd say its more straightforward to raise a custom `NoValidStructures` exception here and handle in with a try/except in the cli

Reply:
I understand your point! nonetheless, if `get_maxint_pdb` gives `None` pdb file we have an if clause that raises a warning, you can test it with `arctic3d Q9Y6A9` for example!